process_nwb/wavelet_transform.py

```diff
@@ -332,8 +332,10 @@ def store_wavelet_transform(elec_series, processing, filters='rat', hg_only=True
         Padding to add to beginning and end of timeseries. Default 0.
     post_resample_rate : float
         If not `None`, resample the computed wavelet amplitudes to this rate.
+    chunked : bool
+        If True, calculate wavelet transform one channel at a time and store iteratively into nwb. Default True
```

Comment:
```suggestion
        If True, calculate wavelet transform one channel and band at a time and store iteratively into nwb. Default True
```

Reply:
Oo I didn't realize it iterates over bands too. Good to know.